envsetup.sh

```diff
@@ -395,7 +414,6 @@ yoe_clean_sstate() {
 
 if [ -z "$DOCKER_REPO" ]; then
   echo "Setting DOCKER_REPO to yoedistro/yoe-build:stretch"
-  echo "set DOCKER_REPO to 'none' to disable docker"
```

Comment:
why delete this ?

Reply:
this is well documented, and was starting to look noisy to me repeated builds. I also think the comment is a bit confusing. Is it displaying something the build system is doing, or is it instructions, etc. I felt it was best to keep instructions in the documentation, and only print status in the log. However, if you want to keep it, I will reword and put back in.

Reply:
documentation is here: https://github.com/YoeDistro/yoe-distro/blob/master/docs/docker.md and in envsetup.sh